dev-docs/modules/relevadRtdProvider.md

```diff
@@ -0,0 +1,118 @@
+| name               | String     | Relevad RTD module name                                      | Mandatory, must be **RelevadRTDModule**                      |
+| waitForIt          | Boolean    | Whether to delay auction for the RTD module response         | Optional. Defaults to false.We recommend setting it to true. Relevad RTD service is very fast. |
+| params             | Object     |                                                              | Relevad RTD module configuration                             |
+| params.partnerid   | String     | Relevad Partner ID, required to enable the service           | Mandatory                                                    |
+| params.publisherid | String     | Relevad publisher id                                         | Mandatory                                                    |
+| params.apikey      | String     | Relevad API key                                              | Mandatory                                                    |
+| param.actualUrl    | String     | Your page URL. When present, will be categorized instead of the browser-provided URL | Optional, defaults to the browser-providedURL                |
+| params.setgpt      | Boolean    | Target or not Google GPT/GAM when it is configured on your page | Optional, defaults to true.                                  |
+| params.minscore    | Integer    | Minimum categorization relevancy score in the range of 0-100. Our categories and segments come  with their relevancy scores. We’ll send to the bidders only categories and segments with the scores higher than the minscore. | Optional, defaults to 30                                     |
+| params.bidders     | Dictionary | Bidders with which to share category and segment information | Optional. If empty or absent, target all bidders.            |
+
+
+
+### Bidder-specific configuration. Every bidder may have these configuration parameters
+
+| Name        | Type             | Description                                                  | Notes                                                    |
+| :---------- | :--------------- | :----------------------------------------------------------- | :------------------------------------------------------- |
+| bidder      | String           | Bidder name                                                  | Mandatory. Example: “appnexus”                           |
+| adUnitCodes | Array of Strings | List of specific AdUnit codes you with to target             | Optional. If empty or absent, all ad units are targeted. |
+| minscore    | Integer          | Bidder-specific minimum categorization relevancy score (0, 100) | Optional, defaults to global minscore above.             |
+
+If you do not have your own `partnerid, publisherid, apikey` please reach out to [anna@relevad.com](mailto:anna@relevad,com).
```

Comment:
typo in the mailto email address

Reply:
@ncolletti - it's not a good idea to use an individual's email address in this or any Prebid doc. You'll forget to update this when Anna leaves.

Reply:
Changed to info@relevad.com
Thank you!